target/linux/ramips/dts/mt7620n_hootoo_tm05.dts

```diff
@@ -0,0 +1,121 @@
+/dts-v1/;
+
+#include "mt7620n.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "hootoo,tm05", "ralink,mt7620n-soc";
+	model = "HooToo TM05";
+
+	aliases {
+		label-mac-device = &ethernet;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wifi {
+			label = "tm05:green:wifi";
+			gpios = <&gpio2 0 GPIO_ACTIVE_HIGH>;
+			linux,default-trigger = "phy0radio";
```

Comment:
Throughput trigger support for MT7620 has been added, so please rebase on latest master and retry with phy0tpt like in this patch:
https://patchwork.ozlabs.org/patch/1216474/

Reply:
Excellent! Will give it a try - I just need to change radio to tpt now (the rest was already done).

Reply:
Confirmed - changed to tpt, and working now. Thanks!

Reply:
Still shows radio for me. Forgot to push?

Reply:
I see it here ... LOL! Will make your other requested changes, force a new push. Thanks!